File: python/15_enums.py
# 15 enums

from enum import Enum, auto

class Status(Enum):
    PENDING = auto()
    RUNNING = auto()
    DONE = auto()
    FAILED = auto()

status = Status.RUNNING
print(status)
print(status.name)
print(status.value)

if status == Status.RUNNING:
    print("still working...")

File: csharp/15_enums.cs
using System;

enum Status {
    Pending,
    Running,
    Done,
    Failed
}

class Program {
    static void Main() {
        Status status = Status.Running;
        Console.WriteLine(status);

        if (status == Status.Running) {
            Console.WriteLine("still working...");
        }
    }
}
